examples/templating/createElement/index.html

```diff
@@ -0,0 +1,35 @@
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="shortcut icon" href="../favicon.png" type="image/x-icon" />
+
+    <link
+      href="https://unpkg.com/tailwindcss@2/dist/tailwind.min.css"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="../app.css" />
+    <link rel="stylesheet" href="../Recommend.css" /> 
+    <title>Algolia JavaScript Recommend - Templating</title>
+  </head>
+
+  <body>
+    <div class="container">
+      <h1>Algolia Recommend</h1>
+      <li class="w-fit hover:underline text-blue-600">
```

Comment:
I'm not sure `w-fit` does anything. Applies to the other places where it's used.

```suggestion
      <li class="hover:underline text-blue-600">
```